example/ecr-pull/main.go

```diff
@@ -27,13 +28,27 @@ import (
 	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
 )
 
+const (
+	defaultParallelism = 0
+)
+
 func main() {
 	ctx := namespaces.NamespaceFromEnv(context.Background())
 
 	if len(os.Args) < 2 {
 		log.G(ctx).Fatal("Must provide image to pull as argument")
 	}
 	ref := os.Args[1]
+	parallelism := defaultParallelism
+	parallelismEnv := os.Getenv("ECR_PULL_PARALLEL")
+	if parallelismEnv != "" {
+		var err error
+		parallelism64, err := strconv.ParseInt(parallelismEnv, 0, 0)
+		if err != nil {
+			log.G(ctx).WithError(err).Fatal("Failed to parse ECR_PULL_PARALLEL")
+		}
+		parallelism = int(parallelism64)
```

Comment:
nit: instead of parsing to int64 and then casting it to int, we could use `Atoi` instead.